linked_list/linked_list.py

```diff
@@ -23,7 +23,14 @@ def __init__(self):
     self.tail = None
 
   def add_to_tail(self, value):
-    pass
+
+    node = Node(value)
+
+    if self.tail is not None:
+      self.tail.set_next(node)
+    else:
+      self.head = node
+    self.tail = node
```

Comment:
Mine looks different than this. I create a new node. I check to see if there's no head, and if there isn't I set the head and tail to the new node. `else` I set the tail's `set_next` to the new node and set the tail itself to the new node.